test/test-cid.spec.js

```diff
@@ -4,8 +4,16 @@
 const base58 = require('bs58')
 const expect = require('chai').expect
 const isIPFS = require('../src/index')
+const CID = require('cids')
 
 describe('ipfs cid', () => {
+  it('isIPFS.cid should match a valid CID instance', (done) => {
+    const cid = new CID('QmYjtig7VJQ6XsnUjqqJvj7QaMcCAwtrgNdahSiFofrE7o')
+    const actual = isIPFS.cid(cid)
+    expect(actual).to.equal(true)
+    done()
+  })
```

Comment:
wow, I can't believe there wasn't a test for this!

Reply:
In past this library operated only on strings (and we had tests for those below), but I feel it makes sense to support instances of `CID` or `Multiaddr` to avoid tedious conversion before calling the check.